src/screens/MainTabs.tsx

```diff
@@ -44,6 +44,7 @@ function MainTabs({navigation}: Props) {
         const [, greenBlockIdToJoin] = matches;
         navigation.navigate('GreenBlock', {
           greenBlockIdToJoin,
+          goTree: false,
```

Comment:
It's not really clear by its name what this param is for. Can we change this to `shouldNavigateToTreeDetails`?

Reply:
yes we can.